GeneratedMonitoringArtifacts/non-default/README.md

```diff
@@ -10,9 +10,9 @@ Azure Monitor metrics addon has pre-built configurations to discover & scrape mo
 
 `kube-api-server` job is turned OFF by default. To collect API-server metrics, do the following -
 
-1. Enable apiserver scraping by specifiying `apiserver = true` under `default-scrape-settings-enabled` in the [settings](https://github.com/Azure/prometheus-collector/blob/main/otelcollector/configmaps/ama-metrics-settings-configmap.yaml) configmap. This will enable scraping apiserver every 30s.
+1. Enable apiserver scraping by specifiying `kube-apiserver = true` under `default-scrape-settings-enabled` in the [settings](https://github.com/Azure/prometheus-collector/blob/main/otelcollector/configmaps/ama-metrics-settings-configmap.yaml) configmap. This will enable scraping apiserver every 30s.
 2. Add more metrics to be collected by the `apiserver` target by specifiying  below -
```

Comment:
I'm not quite sure if I should be updating this file and also the [settings file in this folder](https://github.com/Azure/prometheus-collector/blob/main/GeneratedMonitoringArtifacts/non-default/ama-metrics-settings-configmap.yaml)

Reply:
I think leave it in and we can get AzMon to review. Up to you if you want to do the additional update to that settings file now pre-review, or just comment and ask during PR, meaning you can defer it for now

Reply:
I'll leave it like this for now and continue with it async, I'll ask them if it is ok to add the ccp-metrics settings in here and also need to clarify who is building the dashboards as the other components in the readme have a link to the grafana dashboard file